ampersand-form-view.js

```diff
@@ -68,6 +90,19 @@ assign(FormView.prototype, Events, {
         }
     },
 
+    setValues: function (data) {
+        var value, field;
+        for (var name in data) {
+            if (data.hasOwnProperty(name)) {
+                field = this.getField(name);
```

Comment:
For the sake of reusability and consistency, would it make sense to remove lines 97-101, and replace them with simply
````
value = data[name];
this.setValue(name, value);
````